src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows.mdx

```diff
@@ -246,15 +246,15 @@ The workflows feature is located under the **Alerts & AI** menu.
   The initial limit to the number of workflows you can add is 1000 per account and the initial limit to filter size is 4096 characters per workflow. Please reach out to your account representative if you would like this number to be increased.
 </Callout>
 
-## Workflows log [#workflows-log]
+## Issue Notifications log [#issue-notification-log]
 
-The Workflows Log provides users a detailed record to keep track of all notifications generated by workflows. This information troubleshoots issues, monitors workflow performance, and ensures that notifications are being sent to the correct destinations. As a result, users can quickly identify any issues that may arise and take corrective action as needed.
+The Issue Notification Log provides users a detailed record to keep track of all notifications generated by workflows. This information troubleshoots issues, monitors workflow performance, and ensures that notifications are being sent to the correct destinations. As a result, users can quickly identify any issues that may arise and take corrective action as needed.
 
-To access Workflows Log, go to **[one.newrelic.com > Alerts & AI > Workflows](https://one.newrelic.com/alerts-ai/workflows)**, then click **Workflows Log**.
+To access the Issue Notification Log, go to **[one.newrelic.com > Alerts & AI > Workflows](https://one.newrelic.com/alerts-ai/workflows)**, then click **Issue Notification Log**.
 
-### Navigating the Workflows Log [#navigate-workflows-log]
+### Navigating the Issue Notification Log [#navigate-notification-log]
 
-The workflows log displays the following columns:
+The issue notification log displays the following columns:
```

Comment:
Here we have "issue notification log", in lowercase, while most of the time it's "Issue Notifications Log", in uppercase. I don't know if it's correct, but just in case I add the comment. Other than that, it looks good to me!